src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -103,7 +103,7 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
     }
 
     private boolean isInvalidAddress(Address address) {
-        Pattern p = Pattern.compile("^\\.$|^[a-zA-ZÀ-ž0-9]{1}[a-zA-ZÀ-ž0-9 \\r\\n\\.“”\",’'\\?\\!\\[\\]\\(\\)/£:\\\\_+\\-%&;]{0,}$");
+        Pattern p = Pattern.compile("^\\.$|^[a-zA-ZÀ-ž0-9]{1}.{0,}$");
```

Comment:
how's this tested?

Reply:
We have already test for it givenACaseUpdateEventWithInvalidJointPartyAddressDetails_thenReturnError and givenACaseUpdateEventWithAddressDetails_thenShouldNotReturnError in CaseUpdatedAboutToSubmitHandlerTest.java

Reply:
Added more test